sdk/storage/ci.yml

```diff
@@ -30,7 +30,9 @@ stages:
       LineCoverageTarget: 76
       BranchCoverageTarget: 41
       LiveTestCtestRegex: azure-storage
-      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+      Clouds: Preview
+      SupportedClouds: Preview
+      UnsupportedClouds: Public,Canary,UsGov,China
```

Comment:
This isn't necessary since `SupportedClouds` does not contain the super set either in the defaults or here where you override it.